packages/platforms/netberg/vendor-config/src/python/netberg/__init__.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python
+
+from onl.platform.base import *
+
+class OnlPlatformNetberg(OnlPlatformBase):
```

Comment:
I cannot understand it.
I had checked the file of accton, I cannot find any inherit information in it.
https://github.com/opencomputeproject/OpenNetworkLinux/blob/master/packages/platforms/accton/vendor-config/src/python/accton/__init__.py